.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  plugins: [],
+  rules: {
+    indent: ["error", 2],
+    semi: ["error", "always"],
+    "no-redeclare": "off",
+    "no-console": "off"
+  },
+  env: {
+    browser: true,
+    mocha: true,
+    jquery: true,
```

Comment:
we don't use mocha neither jquery in this repo.

Reply:
Ok, I'll remove these options and re-commit